shared/app.js

```diff
@@ -91,6 +80,35 @@ module.exports = Backbone.Model.extend({
 
   /**
    * @shared
+   *
+   * Initialize the `templateAdapter`, allowing application developers to use whichever
+   * templating system they want.
+   *
+   * We can't use `this.get('templateAdapter')` here because `Backbone.Model`'s
+   * constructor has not yet been called.
+   */
+  initializeTemplateAdapter: function(entryPath, attributes) {
+    if (this.options.templateAdapterInstance) {
+      this.templateAdapter = this.options.templateAdapterInstance;
+    } else {
+      var templateAdapterModule = attributes.templateAdapter || this.defaults.templateAdapter,
+      templateAdapterOptions = {entryPath: entryPath};
+
+      templateAdapterOptions.templateFinder = this.getTemplateFinder();
```

Comment:
I see what you are saying. 
I'm not sure this would break, as this line https://github.com/rendrjs/rendr-handlebars/blob/master/index.js#L11 should just require the default templateFinder if the options we're passing here is set to `_.noop` (i.e undefined).

But what's sure is that being careful won't hurt, so I'll change this as you suggested :)